### ISSUE TITLE
[PHP 8.0] Private methods cannot be final as they are never overridden by other classes

### ISSUE THREAD

**Issue by williamdes** [DASPRiD/Enum#5]
https://travis-ci.org/github/phpmyadmin/phpmyadmin/jobs/718176743#L420

```
PHP Warning:  Private methods cannot be final as they are never overridden by other classes in vendor/dasprid/enum/src/AbstractEnum.php on line 88
PHP Warning:  Private methods cannot be final as they are never overridden by other classes in vendor/dasprid/enum/src/AbstractEnum.php on line 128

Warning: Private methods cannot be final as they are never overridden by other classes in vendor/dasprid/enum/src/AbstractEnum.php on line 88
Warning: Private methods cannot be final as they are never overridden by other classes in vendor/dasprid/enum/src/AbstractEnum.php on line 128
```

**Comment by ausi** [comment]
I created a pull request to fix this: #6

**Comment by williamdes** [comment]
Fixed in 1.0.3
